pkgs/by-name/zo/zoho-mail-desktop/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+appimageTools.wrapType2 rec {
+  pname = "zoho-mail-desktop";
+  version = "1.6.5";
+
+  src = fetchurl {
+    url = "https://downloads.zohocdn.com/zmail-desktop/linux/${pname}-lite-x64-v${version}.AppImage";
+    hash = "sha256-TiPf5tQnVjPpzteSxIa4RLEyYBx6ohnnxPTAG0JjnrM=";
+  };
+
+  extracted = appimageTools.extract { inherit pname version src; };
+
+  extraInstallCommands = ''
+    install -m 444 -D ${extracted}/${pname}.desktop -t $out/share/applications
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace-fail 'Exec=AppRun' 'Exec=${pname}'
```

Comment:
```suggestion
    install -m 444 -D ${extracted}/zoho-mail-desktop.desktop -t $out/share/applications
    substituteInPlace $out/share/applications/zoho-mail-desktop.desktop \
      --replace-fail 'Exec=AppRun' 'Exec=zoho-mail-desktop'
```

Reply:
Just curious. What's the reasoning behind these changes?